### ISSUE TITLE
Game crashes with empty or invalid AllowedCharacters config

### ISSUE THREAD

**Issue by elamec** [Alderon-Games/pot-community-servers#70]
Client crashes if AllowedCharacters is set to invalid value or left empty ("AllowedCharacters=") on the server.

Please fix it to enable empty/none value and let it result that none of the characters is available on the server.
So that if spectator mode is assigned to players, the server can be used only in spectator mode (like the former AI spectator server).

**Comment by AlderonEmu** [comment]
> Please fix it to enable empty/none value and let it result that none of the characters is available on the server.
> So that if spectator mode is assigned to players, the server can be used only in spectator mode (like the former AI spectator server).

Can you explain this further, please?

**Comment by elamec** [comment]
TL;DR:
I don't need this feature currently + in 26361 it's possible to create a spectator only server so I close the ticket now.
Thanks for your support!

--
Background:
I wanted to create a server where you can only enter as a spectator, like it worked for the old AI spectator server (https://youtu.be/PJE4VrvAWFM?t=860)

I used this config in Game.ini:
    [/Script/PathOfTitans.IGameSession]
    ...
    AllowedCharacters=

When I started the servre with this config, it immediately crashed once someone wanted to enter it.
But I've checked it now (26361) and currently it works 👍 
https://youtu.be/GI_6Vtf8KV4

Thanks very much!
